examples/cpVerifyFlat.ipynb

```diff
@@ -19,145 +27,121 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 12,
    "id": "84dccd11",
    "metadata": {
-    "scrolled": true
+    "execution": {
+     "iopub.execute_input": "2023-01-13T22:34:10.757323Z",
+     "iopub.status.busy": "2023-01-13T22:34:10.756558Z",
+     "iopub.status.idle": "2023-01-13T22:34:10.781519Z",
+     "shell.execute_reply": "2023-01-13T22:34:10.781039Z",
+     "shell.execute_reply.started": "2023-01-13T22:34:10.757299Z"
+    },
+    "scrolled": true,
+    "tags": []
    },
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "/home/czw/.local/lib/python3.8/site-packages/ginga/cmap.py:13317: MatplotlibDeprecationWarning: The global colormaps dictionary is no longer considered public API.\n",
-      "  for name in _cm.cmap_d:\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
     "# This cell should be edited to match the data to be inspected.\n",
     "\n",
     "# Which calibration type to analyse.\n",
     "calibType = 'flat'\n",
-    "physical_filter = 'empty~empty'\n",
+    "physical_filter = 'BG40_65mm_1~empty'\n",
     "# Camera\n",
     "cameraName = 'LATISS'\n",
     "\n",
     "# Set which display to use.\n",
     "afwDisplay.setDefaultBackend(\"astrowidgets\")\n",
```

Comment:
Astrowidgets don't work at the USDF RSP, for example. Should this be in a try block with an alternative display in case of failure?

Reply:
I've added a try, and asserts to the cells the use that display so they are skipped.  I've also added https://jira.lsstcorp.org/browse/DM-37592 to cover the documentation aspects you bring up.